.gitignore

```diff
@@ -28,3 +25,5 @@ db/*.csv
 .coverage
 # ESLint cache
 .eslintcache
+# Logging text files
+*.txt
```

Comment:
Technically this will also ignore `requirements.txt` and `runtime.txt`, although they won't be ignored because they were created before the change was made. I think a better practice is to specifically prevent ignoring those files (with leading exclamation marks) or ignore specific log files (e.g. `log/*.txt`).